include/redisserver.h

```diff
@@ -119,10 +154,11 @@ class RedisServer {
         /*!
         *   \brief Get a Tensor from the server
         *   \param key The name of the tensor to retrieve
+        *   \param cmd The command to run on the server
         *   \returns The CommandReply from the get tensor server
         *            command execution
         */
-        virtual CommandReply get_tensor(const std::string& key) = 0;
+        virtual CommandReply get_tensor(const std::string& key, GetTensorCommand& cmd) = 0;
```

Comment:
I'm not sure that we need to pass the ``GetTensorCommand`` type down to ``RedisServer`` from ``Client``.  I think we can probably just construct the ``GetTensorCommand`` inside of ``RedisServer::get_tensor()`` if we make the parsing function a static member variable (i.e. we won't need the object only the class at the ``Client`` level to parse the output).